src/main/java/com/aws/greengrass/logmanager/util/Digest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.logmanager.util;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Base64;
+import com.aws.greengrass.util.Utils;
+
+/**
+ * Set of methods for LogLine digest
+ */
+
+public class Digest {
```

Comment:
Why are you adding this? 

This already exists in Greengrass, you can just use it.

Reply:
Good point. Taking this commit back.